### PR TITLE
Update mirrors.json

### PR DESCRIPTION
**Pull request by fhdk**
splitted belnet.be - different paths for http and ftp
removed - ftp - no route to host - from japanese mirror
removed http from uex.dk - I use only https

### CODE REVIEW

File: mirrors.json [hunk 1]
```diff
@@ -38,12 +38,18 @@
         "country": "Belgium",
         "url": "http://ftp.belnet.be/manjaro/",
         "protocols": [
-            "ftp",
             "http"
         ]
     },
     {
         "country": "Belgium",
+        "url": "http://ftp.belnet.be/mirrors/manjaro/",
```
Comment (philmmanjaro) [inline mirrors.json]: Isn't it `ftp://ftp.belnet.be/mirrors/manjaro/`

Comment (fhdk) [inline mirrors.json]: It is a typo - you're right

**Comment by Huluti** [comment]
Is it ok for you @philmmanjaro?

**Comment by philmmanjaro** [comment]
@Huluti: seems to be fine now.

**Pull request merged**